src/main/java/com/uber/jenkins/phabricator/tasks/NonDifferentialHarbormasterTask.java

```diff
@@ -81,13 +87,15 @@ protected void setup() {
      */
     @Override
     protected void execute() {
-        final boolean pass = buildResult.isBetterOrEqualTo(hudson.model.Result.SUCCESS);
-        info(String.format("Sending diffusion result as: %s", buildResult.toString()));
+        final MessageType messageType = this.sendPartialResults ?
```

Comment:
We should convert it into simple condition, it is quite crucial logic hidden in ternary operator

Reply:
SGTM. Cleaned up here and the other instance of this logic.